app/lib/services/restApi.ts

```
@@ -1215,3 +1216,17 @@ export const getUsersRoles = async (): Promise<boolean | IRoleUser[]> => {
 
 export const getSupportedVersionsCloud = (uniqueId?: string, domain?: string) =>
 	fetch(`https://releases.rocket.chat/v2/server/supportedVersions?uniqueId=${uniqueId}&domain=${domain}&source=mobile`);
+
+export const mediaCallsStateSignals = async (
+	contractId: string
+): Promise<{ signals: ServerMediaSignal[]; success: boolean }> => {
```

ESLint

[error] Replace `⏎↹contractId:·string⏎` with `contractId:·string`

(prettier/prettier)